### PR TITLE
Bump capstone to 1995ddf9

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: test/db/tools/rz [hunk 1]
```diff
@@ -141,6 +141,7 @@ FILE==
 CMDS=!rizin -R ""
 EXPECT_ERR=<<EOF
 EOF
+BROKEN=1
```
Comment (XVilka) [inline test/db/tools/rz:144]: I wonder why this got broken?

**Review by XVilka — APPROVED** [review]

File: test/db/tools/rz (hunk 1 above)
Comment (b1llow) [reply test/db/tools/rz:144]: I remember because of the extra log line

**Review by wargio — APPROVED** [review]

**Review by Rot127 — APPROVED** [review]

**Review by XVilka — CHANGES_REQUESTED** [review]
I think it's probably better to merge Tricore capstone changes into one commit (to override old syntax changes) in the capstone-ng directly, also rebase the branch itself. Feel free to do that directly. Please also use the latest available capstone `next` commit: https://github.com/capstone-engine/capstone/commits/next

**Comment by b1llow** [comment]
> I think it's probably better to merge Tricore capstone changes into one commit (to override old syntax changes) in the capstone-ng directly, also rebase the branch itself. Feel free to do that directly. Please also use the latest available capstone `next` commit: https://github.com/capstone-engine/capstone/commits/next

done

**Pull request closed without merging**